src/vorta/autostart.py

```diff
@@ -0,0 +1,71 @@
+import sys
+from pathlib import Path
+from PyQt5 import QtCore
+from setuptools import Distribution
+from setuptools.command.install import install
+
+
+GNOME_STARTUP_FILE = """[Desktop Entry]
```

Comment:
style tip:

foo = """\
first line
second
...
"""

Reply:
Nice, I will remember that one!

Reply:
Changed. Good tip.